### PR TITLE
Prepare v1.6.1 release

### PR DESCRIPTION
**Pull request by ThomsonTan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,12 +15,27 @@ Increment the:
...
 
+## [1.6.1] 2022-09-22
+
 * [BUILD] Upgrade opentelemetry-proto to v0.19.0 [#1579](https://github.com/open-telemetry/opentelemetry-cpp/pull/1579)
 * [METRICS EXPORTER] Add `OtlpGrpcMetricExporterFactory` and `OtlpHttpMetricExporterFactory`.
   [#1606](https://github.com/open-telemetry/opentelemetry-cpp/pull/1606)
-* [METRICS EXPORTER] Add `OtlpGrpcClient` [#1606](https://github.com/open-telemetry/opentelemetry-cpp/pull/1606)
-* [BUILD] Fix header only api singletons [#1604](https://github.com/open-telemetry/opentelemetry-cpp/pull/1604)
 * [SEMANTIC CONVENTIONS] Upgrade to version 1.13.0 [#1624](https://github.com/open-telemetry/opentelemetry-cpp/pull/1624)
+* [BUILD] Fixes issue #1612. [#1622](https://github.com/open-telemetry/opentelemetry-cpp/pull/1622)
```
Comment (lalitb) [inline CHANGELOG.md:24]:
```suggestion
* [BUILD] Fixes span creation benchmark issue. [#1622](https://github.com/open-telemetry/opentelemetry-cpp/pull/1622)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,12 +15,27 @@ Increment the:
...
+* [METRICS SDK] Fix default value of `OtlpHttpMetricExporterOptions::aggregation_temporality`. [#1601](https://github.com/open-telemetry/opentelemetry-cpp/pull/1601)
+* [METRICS EXAMPLE] Example for OTLP gRPC exporter for Metrics. [#1598](https://github.com/open-telemetry/opentelemetry-cpp/pull/1598)
+* [SDK] Fix `LoggerContext::Shutdown` and tsan of `OtlpHttpClient` [#1592](https://github.com/open-telemetry/opentelemetry-cpp/pull/1592)
+* [METRICS SDK] Fix 1585 - Multiple cumulative metric collections without measurement recording. [#1586](https://github.com/open-telemetry/opentelemetry-cpp/pull/1586)
+* [BUILD] metrics warnings [#1583](https://github.com/open-telemetry/opentelemetry-cpp/pull/1583)
+* [METRICS SDK] Fix ObservableInstrument::RemoveCallback [#1582](https://github.com/open-telemetry/opentelemetry-cpp/pull/1582)
+* [SDK] Add error log when getting a http error code [#1581](https://github.com/open-telemetry/opentelemetry-cpp/pull/1581)
+* [EXPORTER] ETW Exporter - Add support for Sampler and ID Generator [#1547](https://github.com/open-telemetry/opentelemetry-cpp/pull/1547)
```
Comment (lalitb) [inline CHANGELOG.md:38]: Should we mention a note in the end, specifying breaking change in the Semantic Convention even though it is experimental, to make people know that they may have to modify their instrumentation library if using these.

**Review by lalitb — APPROVED** [review]

**Review by marcalff — APPROVED** [review]
LGTM

**Review by esigo — APPROVED** [review]
LGTM
Thanks :)

**Review by owent — APPROVED** [review]
LGTM

**Pull request merged**